routes/web.php

```diff
@@ -1,4 +1,4 @@
-<?php
```

Comment:
This needs to be added... it's causing the entire branch to fail. PHP files are required to have the `<?php` directive